netcode.c

```diff
@@ -4163,7 +4207,12 @@ void netcode_server_connect_client( struct netcode_server_t * server,
     char address_string[NETCODE_MAX_ADDRESS_STRING_LENGTH];
 
     /* TODO: try to find a way to deliver match itd. */
-    if ( !skillz_add_client_to_match( server, 111, client_id ) )
+    int testId = 0;
+    if( server->num_connected_clients >= 3 )
```

Comment:
Nit but it's a best practice to put brackets around if/else statements (future-proofing thing)

Reply:
are we ever going to have more than 2 connected clients?

Reply:
bump

Reply:
Yes, this allows for 4 for testing.  There will be lots more.

Reply:
Or do you mean more than 2 clients connected to a match?

Reply:
That is what I meant.  AFAIK we're only required to support 2 for this release.